spec/unit/ssl/state_machine_spec.rb

```diff
@@ -7,12 +7,12 @@
   include PuppetSpec::Files
 
   let(:machine) { described_class.new }
-  let(:cacert_pem) { File.read(fixtures('unit/ssl/ssl_provider/ca.pem')) }
-  let(:cacert) { OpenSSL::X509::Certificate.new(cacert_pem) }
+  let(:cacert_pem) { cacert.to_pem }
+  let(:cacert) { cert_fixture('ca.pem') }
   let(:cacerts) { [cacert] }
 
-  let(:crl_pem) { File.read(fixtures('unit/ssl/ssl_provider/crl.pem')) }
-  let(:crl) { OpenSSL::X509::CRL.new(crl_pem) }
+  let(:crl_pem) { crl.to_pem }
+  let(:crl) { crl_fixture('crl.pem') }
```

Comment:
Same, it looks like this won't work, since we're using a value on line 14 that's defined on line 15.... unless I'm missing something?